### PR TITLE
WIP: [DO NOT MERGE] Investiage risk reprocessing in SAC tests

### PR DESCRIPTION
**Pull request by dhaus67**
## Description

A detailed explanation of the changes in your PR.

Feel free to remove this section if it is overkill for your PR, and the title of your PR is sufficiently descriptive.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

TODO(replace-me)
Use this space to explain how you tested your PR, or, if you didn't test it, why
you did not do so. Valid reasons include, for example, "CI is sufficient",
"No testable changes". Feel free to attach JSON snippets, curl commands,
screenshots.

In addition to reviewing your code, reviewers **must** also review your testing
instructions and make sure they are sufficient.

### CODE REVIEW

**Comment by dhaus67** [comment]
Current dependencies on/for this PR:
* master
  * **PR #2661** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/2661" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/stackrox/stackrox/2661?utm_source=stack-comment).

File: qa-tests-backend/scripts/run-part-1.sh [hunk 1]
```diff
@@ -58,38 +58,21 @@ get_ECR_docker_pull_password() {
...
-        info "An unexpected context. Defaulting to BAT tests only..."
-        make -C qa-tests-backend bat-test || touch FAIL
-    fi
+    info "Running SAC tests 100 times, exiting on the first failure found"
 
+    for i in {1..100}
+    do
+      info "SAC test count: ${i}"
+      make -C qa-tests-backend sac-test || touch FAIL
+      [[ ! f FAIL ]] || store_qa_test_results "part-1-tests" && die "SAC test failed"
```
Comment (dhaus67) [inline qa-tests-backend/scripts/run-part-1.sh:73]:
```suggestion
      [[ ! -f FAIL ]] || store_qa_test_results "part-1-tests" && die "SAC test failed"
```

**Comment by janisz** [comment]
I think we can close it in favour of https://github.com/stackrox/stackrox/pull/2992

**Pull request closed without merging**